### DISCUSSION TITLE
Selectize field (not dropdown) max height

### DISCUSSION THREAD

**Discussion by reformed** [selectize/selectize.js#1736] [Q&A] [answered]
Is there a way to limit the max height of a Selectize field? The field itself, not the dropdown. The field in question needs to allow all options to be selected, so the option `maxItems` cannot be used. The `maxOptions` option can't be used either because that applies to the dropdown.

Is there a way to reduce the height of a field like this?

![Screenshot of example](https://i.stack.imgur.com/OcGvM.png)

Example:

```html
    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.13.3/css/selectize.min.css">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.13.3/css/selectize.bootstrap3.min.css" rel="stylesheet" />

    <div id="container" style="max-width: 400px;">
      <form>
        <select id="selectize" name="selectize" class="form-control" multiple></select>
        <button type="submit" class="btn btn-success">Submit</button>
      </form>
    </div>

    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/selectize.js/0.13.3/js/standalone/selectize.min.js"></script>
    <script>
        (function($select) {
          for (let i = 100; i < 202; ++i) {
            $select.append('<option value="' + i + '" selected>Test ' + i + '</option>');
            $select.append('<option value="' + (2 * i) + '">Test ' + (2 * i) + '</option>');
          }

          $select.selectize();
        })($("#selectize"));
    </script>
```

**Comment by risadams** [comment]
If I am following the question you can use CSS to add a max height on the .selectize-input class.
An example is here: https://codepen.io/risadams/pen/WNXeJjK

**Reply by reformed** [reply]
This approach makes an "at a glance" view of what has already been selected extremely difficult. You can only see the last 2 rows or so of selected items, and scrolling does nothing. In order to view some of the earlier selections, you have to continuously press the left arrow key.

I added a div container around the `select` element with a `max-height` and `overflow-y`, then set the Selectize option `dropdownParent` to "body". This seems to solve both problems: [https://codepen.io/reformed05/pen/PoOYeye](https://codepen.io/reformed05/pen/PoOYeye)

**Comment by reformed** [comment, marked as answer]
Wrapping the `select` field in a container with CSS `max-height` and `overflow-y` set to "auto", and then setting the Selectize option `dropdownParent` to "body" seems to solve the problem. Example: [https://codepen.io/reformed05/pen/PoOYeye](https://codepen.io/reformed05/pen/PoOYeye)
